api/lib/router/routes/admin/admin-checkout.ts

```diff
@@ -67,6 +68,28 @@ export class AdminCheckoutController extends ScannerController implements IExpre
       ),
       (req, res, next) => this.returnObjectHandler(req, res, next),
     );
+    // Create a checkout item
+    app.post(
+      '/items',
+      (req, res, next) => this.verifyScannerPermissionsMiddleware(
```

Comment:
I think this one should just refer to the normal authService, since I don't think we're expecting scanners to add new items to the inventory